ipam/actor.go

```diff
@@ -0,0 +1,165 @@
+package ipam
```

Comment:
As this was mostly boilerplate we chose to keep it out of allocator.go; this seems pretty idiomatic for a go PoV.  It keeps allocator.go more manageable and understandable.  Not as relevant now we don't have all those structs & type switches, but I still think it counts.  Do you have any rules as to when it is appropriate to spit large files?